flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -833,10 +832,44 @@ public TableResultInternal executeInternal(List<ModifyOperation> operations) {
         return executeInternal(transformations, sinkIdentifierNames, jobStatusHookList);
     }
 
-    private ModifyOperation getOperation(ReplaceTableAsOperation rtasOperation) {
-        // rtas drop table first, then create
+    private ModifyOperation getModifyOperation(
+            ReplaceTableAsOperation rtasOperation, List<JobStatusHook> jobStatusHookList) {
         CreateTableOperation createTableOperation = rtasOperation.getCreateTableOperation();
```

Comment:
Get warning from my IDE
`Duplicated code fragment (7 lines long) `

Reply:
Similar to part of the CTAS code